scripts/lib/build-pgroonga

```diff
@@ -4,9 +4,6 @@ set -e
 
 PGROONGA_VERSION="2.1.6"
 
-# Currently only tested for Fedora systems.
-
-yum install -y groonga-devel msgpack-devel
```

Comment:
If this is removed then the script is no longer self-contained. If so, might add a mechanism that exits the script early if it is called directly.